### PR TITLE
search improvements

### PR DESCRIPTION
**Pull request by echjordan**
https://electron-website-pr-1248.herokuapp.com

### CODE REVIEW

**Comment by zeke** [comment]
Not sure why the Heroku deploy didn't work. I will dig into that.

Heads up: It looks like CI is now failing because of a change in the HTML structure:

```
  1) localized views
       every localized string is used in a view (except for pages.* and _404.*):
     AssertionError: expected [ Array(100) ] to include 'nav.search'
```

**Comment by zeke** [comment]
[status.heroku.com](http://status.heroku.com/) says Heroku has the hiccups. I pushed up an empty commit to trigger another deployment.

**Comment by MarshallOfSound** [comment]
Looks :awesome: so far 😀 Just wanted to drop by and say it would be 💯 if you could prefix searches with things to restrict results.

E.g. `doc: ` to limit searches to just the API documentation or `app: ` to search for apps 🤔 

Normally you would want to search for one or the other so narrowing that down would be cool 😄

File: scripts/search.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+  function showHits () {
+    document.getElementById('hits').style.display = 'flex'
+  }
+
+  function hideHits () {
+    document.getElementById('hits').style.display = 'none'
+  }
+
+  function checkIfChild (parentElement, checkElement) {
```
Comment (echjordan) [inline scripts/search.js]: @zeke HALP. trying to recursively check for any child of "hits" to fix click functionality. something is broken. Line 135 calls this function.

**Pull request merged**